src/index.js

```diff
@@ -1,13 +1,35 @@
-import _ from 'lodash';
 import './style.css';
 
-// function component() {
-//     const element = document.createElement('div');
-  
-//     // Lodash, currently included via a script, is required for this line to work
-//     element.innerHTML = _.join(['Hello', 'mustafa'], ' ');
-  
-//     return element;
-//   }
-  
-//   document.body.appendChild(component());
\ No newline at end of file
+const listGroup = document.querySelector('.list-group');
+const ListArray = [
+  { id: 1, description: '1111111111111', isCompleted: false },
+  { id: 2, description: '2222222222222', isCompleted: false },
+  { id: 3, description: '3333333333333', isCompleted: false },
```

Comment:
[OPTIONAL]

- Always give meaningful task description to keep the app meaningful.